php/WP_CLI/CommandWithMeta.php

```diff
@@ -77,6 +77,15 @@ public function list_( $args, $assoc_args ) {
 	/**
 	 * Get meta field value.
 	 *
+	 * <id>
+	 * : The ID of the object.
+	 *
+	 * <key>
+	 * : The name of the meta field to get.
+	 *
+	 * [--format=<format>]
+	 * : Accepted values: table, json. Default: table
+	 *
 	 * @synopsis <id> <key> [--format=<format>]
```

Comment:
Can you remove `@synopsis` ? Now that we have expanded docs here, it's no longer necessary.